vagrant-post-script/orchestrate_docker.sh

```diff
@@ -1,8 +1,4 @@
 #!/bin/bash
 
 cd oms-docker/docker
-docker-compose up -d
-echo "############# ok until here"
-docker-compose logs -f omscore-bootstrap
-docker-compose logs -f omsevents-bootstrap
-echo "############# done"
+make bootstrap
```

Comment:
The Vagrant machine is just a standard dev env for everyone - just an ubuntu. Mainly for windows and mac people, but linux people should also use it only to avoid situations of "works on mine only" because that particular version of that particular library of that moment in time when it was installed was compiled in a certain way.. that's the rationale. 

Also soon the Vagrantfile will contain 2 VMs to swarm it